__tests__/app.test.js

```diff
@@ -75,3 +75,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("Status: 200, returns an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
+      });
+  });
+  test("Status: 200, returns an array of length 4", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+      });
+  });
```

Comment:
You essentially cover these two in the third test. 
You check the length on line 102, and could just put the array expectation (line 85) inside the test below. 
The fact you can use a forEach on it also tells you it's an array anyway. 

Possibly redundant tests